modules/framework/galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/resources/StreamBean.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.common.resources;
+
+import javax.servlet.http.HttpServletResponse;
+
+import dev.galasa.framework.api.common.InternalServletException;
+import dev.galasa.framework.api.common.ServletError;
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.IConfigurationPropertyStoreService;
+import static dev.galasa.framework.api.common.ServletErrorMessage.*;
+
+public class StreamBean {
```

Comment:
Is this class needed? Could we not just use the streams service and delete the stream that way?